tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/metrics/AllMetricsTest.java

```diff
@@ -79,6 +87,10 @@ public static WebArchive deploy() {
     @Inject
     private AllMetricsBean allMetricsBean;
     
+    @Inject
+    @RegistryType(type = Type.BASE)
```

Comment:
This made me think: 
Do we have test to verify the metrics is under base instead of application?

Reply:
We don't have an explicit test, but reading the Metrics API docs, I thought that the registries were separate and it wouldn't give you an aggregated view. When you inject a `MetricsRegistry` without specifying the type, you get the application registry.

In our tests we specify that we want the base registry, both here and in `MetricGetter`

Reply:
The only other test we could add would be to ensure that metrics are _not_ registered under the application scope. I'm not sure we should add such a test since it wouldn't